### PR TITLE
adding a function responsible for creating addition and subtraction e…

### PR DESCRIPTION
**Pull request by Frexor**
adding quations and saving them in array

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -134,4 +134,22 @@ static void drawEqual(BufferedImage img, int offset) {
         g2d.fillRect(9 + offset, 89, 82, 2);
     }
 
+    static void EquationResult(String[] args) {
```
Comment (piekaa) [inline src/Main.java]: String[] args parameter is not used.
method should return array of Strings, not void

File: src/Main.java [hunk 2]
```diff
@@ -134,4 +134,22 @@ static void drawEqual(BufferedImage img, int offset) {
...
+                if (i + j < 10) {
+                    result[index++] = String.format("%d + %d = %d", i, j, i + j);
+                }
+                if (i - j > -1) {
+                    result[index++] = String.format("%d - %d = %d", i, j, i - j);
+                }
+            }
+        }
+        for (int i = 0; i < 110; i++) {
+            System.out.println(result[i]);
```
Comment (piekaa) [inline src/Main.java]: no need to print

File: src/Main.java [hunk 3]
```diff
@@ -134,4 +134,22 @@ static void drawEqual(BufferedImage img, int offset) {
         g2d.fillRect(9 + offset, 89, 82, 2);
     }
 
+    static void EquationResult(String[] args) {
+        String result[] = new String[110];
+        int index = 0;
+        for (int i = 0; i <= 9; i++) {
```
Comment (piekaa) [inline src/Main.java]: < 10 would look nicer, but it's up to you, same line below

File: src/Main.java [hunk 4]
```diff
@@ -134,4 +134,22 @@ static void drawEqual(BufferedImage img, int offset) {
...
 
+    static void EquationResult(String[] args) {
+        String result[] = new String[110];
+        int index = 0;
+        for (int i = 0; i <= 9; i++) {
+            for (int j = 0; j <= 9; j++) {
+                if (i + j < 10) {
+                    result[index++] = String.format("%d + %d = %d", i, j, i + j);
+                }
+                if (i - j > -1) {
```
Comment (piekaa) [inline src/Main.java]: zero or more sounds better than more than minus one, thus I think >= 0 is better here.

File: src/Main.java (hunk 1 above)
Comment (piekaa) [inline src/Main.java]: method name should start with lowercase

**Review by piekaa — CHANGES_REQUESTED** [review]
Some changes are required

File: src/Main.java [hunk 5]
```diff
@@ -134,4 +134,22 @@ static void drawEqual(BufferedImage img, int offset) {
         g2d.fillRect(9 + offset, 89, 82, 2);
     }
 
+    static void EquationResult(String[] args) {
+        String result[] = new String[110];
+        int index = 0;
+        for (int i = 0; i <= 9; i++) {
+            for (int j = 0; j <= 9; j++) {
+                if (i + j < 10) {
+                    result[index++] = String.format("%d + %d = %d", i, j, i + j);
```
Comment (piekaa) [inline src/Main.java]: Spaces between numbers and signs not needed.

**Review by piekaa — CHANGES_REQUESTED** [review]

File: src/Main.java (hunk 1 above)
Comment (Frexor) [reply src/Main.java]: Done. Change `static void equationResult(String[] args)` to `static String[] equationResult(String[] args)`.

File: src/Main.java (hunk 2 above)
Comment (Frexor) [reply src/Main.java]: Done as suggested.

File: src/Main.java (hunk 3 above)
Comment (Frexor) [reply src/Main.java]: Done as suggested. 
Change:
```
for (int j = 0; j <= 9; j++) {
    if (i + j <= 9) {
```
to:
```
for (int j = 0; j < 10; j++) {
    if (i + j < 10) {
```

File: src/Main.java (hunk 4 above)
Comment (Frexor) [reply src/Main.java]: Done, change` if (i - j > -1)` to `if (i - j >= 0)`

File: src/Main.java (hunk 1 above)
Comment (Frexor) [reply src/Main.java]: Done as suggested.

File: src/Main.java (hunk 5 above)
Comment (Frexor) [reply src/Main.java]: Done, change `result[index++] = String.format("%d + %d = %d", i, j, i + j)` to `result[index++] = String.format("%d-%d=%d", i, j, i - j)`

File: src/Main.java [hunk 6]
```diff
@@ -134,4 +134,19 @@ static void drawEqual(BufferedImage img, int offset) {
         g2d.fillRect(9 + offset, 89, 82, 2);
     }
 
+    static String[] equationResult(String[] args) {
```
Comment (piekaa) [inline src/Main.java]: again String[] args not needed

**Review by piekaa — CHANGES_REQUESTED** [review]

File: src/Main.java (hunk 6 above)
Comment (Frexor) [reply src/Main.java]: Done.

**Review by piekaa — APPROVED** [review]
debesta

**Pull request merged**